tigerpath/majors_and_certificates/majors/COM_2018.json

```
@@ -0,0 +1,263 @@
+      "double_counting_allowed": false,
+      "req_list": [
+        {
+          "name": "Comparative Literature",
+          "max_counted": 1,
+          "min_needed": 2,
+          "explanation": "All majors beginning with the class of 2021, must take COM 300 (the Junior Seminar), and two other COM courses taught by core COM faculty (that is, by faculty listed on the “Core Faculty” list of the departmental website). Majors must take COM 300 in the fall term of their junior year, unless they are studying abroad then, in which case they must take the course in their senior year instead. COM has also sometimes accepted the English-department junior seminar taught in London by Princeton faculty. The other two COM courses can be 200-, 300-, 400-, or 500-level; with the professor’s permission, advanced students may enroll in 500-level COM courses. COM majors in the classes of 2019 and 2020 fulfill this requirement by taking COM 300 (the Junior Seminar), and any other COM courses or courses cross-listed with COM. These must be Princeton COM courses; no exceptions.",
+          "excluded_course_list": [
+            "COM 300"
+          ],
+          "course_list": [
+            "COM *"
+          ]
+        },
+        {
+          "name": "Junior Seminar",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "All majors beginning with the class of 2021, must take COM 300 (the Junior Seminar), and two other COM courses taught by core COM faculty (that is, by faculty listed on the “Core Faculty” list of the departmental website). Majors must take COM 300 in the fall term of their junior year, unless they are studying abroad then, in which case they must take the course in their senior year instead. COM has also sometimes accepted the English-department junior seminar taught in London by Princeton faculty. The other two COM courses can be 200-, 300-, 400-, or 500-level; with the professor’s permission, advanced students may enroll in 500-level COM courses. COM majors in the classes of 2019 and 2020 fulfill this requirement by taking COM 300 (the Junior Seminar), and any other COM courses or courses cross-listed with COM. These must be Princeton COM courses; no exceptions.",
+          "course_list": [
+            "COM 300"
+          ]
+        },
+        {
+          "name": "Non-English-Languge Reading",
+          "max_counted": 1,
+          "min_needed": 3,

```

Shouldn't this be 4?
```suggestion
          "min_needed": 4,
```